index.bs

```diff
@@ -5297,14 +5301,12 @@ created [=/request=] belongs to is [=transaction/aborted=] using the steps to
 
 
 <!-- ============================================================ -->
-## Running an upgrade transaction ## {#upgrade-transaction-steps}
+## Upgrading a database ## {#upgrade-transaction-steps}
 <!-- ============================================================ -->
 
 <div algorithm>
 
-To <dfn>run an upgrade transaction</dfn> with |connection| (a [=/connection=])
-which is used to update the [=/database=], a new |version| to be set
-for the [=/database=], and a |request|, run these steps:
+To <dfn>upgrade a database</dfn> with |connection| (a [=/connection=]), a new |version|, and a |request|, run these steps:
```

Comment:
Do you think it's worthwhile to preserve the old ID? There probably don't exist many references to https://www.w3.org/TR/IndexedDB/#run-an-upgrade-transaction out on the interwebs, but if there are they'd be broken

Reply:
Good question. After a bit of searching I don't see many references to the old phrase out on the webs, so I'm suspecting the number of direct links into the algorithm from anywhere other than issues in this repo are low. So... leaning to "no" but we can restore the id if anyone complains?